package/MDAnalysis/lib/pkdtree.py

```diff
@@ -127,22 +130,22 @@ def set_coords(self, coords, cutoff=None):
 
         """
         # If no cutoff distance is provided but PBC aware
-        if self.pbc and (cutoff is None):
-            raise RuntimeError('Provide a cutoff distance'
-                               ' with tree.set_coords(...)')
 
         # set coords dtype to float32
         # augment coordinates will work only with float32
         coords = np.asarray(coords, dtype=np.float32)
 
         if self.pbc:
-            self.cutoff = cutoff
```

Comment:
This line is extremely important and cannot be removed! This line actually sets the cutoff in the object; without it, `self.cutoff` will always be `None`. The line needs to be run if the exception about the cutoff is not raised.